src/mlpack/methods/ann/layer/convolution.hpp

```diff
@@ -259,6 +259,12 @@ class Convolution
     return (outSize * inSize * kernelWidth * kernelHeight) + outSize;
   }
 
+  //! Get the shape of the input.
+  size_t InputShape() const
+  {
+    return inputHeight*inputWidth*inSize;
```

Comment:
```suggestion
    return inputHeight * inputWidth * inSize;
```

This is just to match the mlpack style guide.  Do you think you can make this change (horizontal whitespace) wherever it's necessary in this PR?

Reply:
Sure, I will make the changes 👍